packages/thumbnail/src/spectrum-thumbnail.css

```diff
@@ -12,353 +12,431 @@ governing permissions and limitations under the License.
+                    ) +
+                    var(
+                        --mod-thumbnail-focus-indicator-thickness,
+                        var(--spectrum-thumbnail-focus-indicator-thickness)
+                    )
+            ) * -1
     );
-    --spectrum-thumbnail-border-radius: var(
-        --spectrum-thumbnail-m-border-radius,
-        var(--spectrum-alias-thumbnail-border-radius-small)
+    inset-block-start: calc(
+        (
+                var(
+                        --mod-thumbnail-focus-indicator-gap,
+                        var(--spectrum-thumbnail-focus-indicator-gap)
+                    ) +
+                    var(
+                        --mod-thumbnail-focus-indicator-thickness,
+                        var(--spectrum-thumbnail-focus-indicator-thickness)
+                    )
+            ) * -1
     );
-    --spectrum-thumbnail-width: var(
-        --spectrum-thumbnail-m-width,
-        var(--spectrum-global-dimension-size-500)
+    inset-inline-end: calc(
+        (
+                var(
+                        --mod-thumbnail-focus-indicator-gap,
+                        var(--spectrum-thumbnail-focus-indicator-gap)
+                    ) +
+                    var(
+                        --mod-thumbnail-focus-indicator-thickness,
+                        var(--spectrum-thumbnail-focus-indicator-thickness)
+                    )
+            ) * -1
     );
-    --spectrum-thumbnail-height: var(
-        --spectrum-thumbnail-m-height,
-        var(--spectrum-global-dimension-size-500)
+    inset-inline-start: calc(
+        (
+                var(
+                        --mod-thumbnail-focus-indicator-gap,
+                        var(--spectrum-thumbnail-focus-indicator-gap)
+                    ) +
+                    var(
+                        --mod-thumbnail-focus-indicator-thickness,
+                        var(--spectrum-thumbnail-focus-indicator-thickness)
+                    )
+            ) * -1
     );
+    position: absolute;
 }
-:host([size='l']) {
-    --spectrum-thumbnail-border-color-selected: var(
-        --spectrum-thumbnail-l-border-color-selected,
-        var(--spectrum-alias-border-color-selected)
+:host(.focus-visible) .image-wrapper,
+:host([focused]) .image-wrapper {
+    border-radius: var(
+        --mod-thumbnail-border-radius,
+        var(--spectrum-thumbnail-border-radius)
     );
-    --spectrum-thumbnail-border-size-selected-key-focus: var(
-        --spectrum-thumbnail-l-border-size-selected-key-focus,
-        var(--spectrum-alias-border-size-thick)
+    overflow: hidden;
+}
+:host(:focus-visible) .image-wrapper,
+:host([focused]) .image-wrapper {
+    border-radius: var(
+        --mod-thumbnail-border-radius,
+        var(--spectrum-thumbnail-border-radius)
     );
-    --spectrum-thumbnail-border-size: var(
-        --spectrum-thumbnail-l-border-size,
-        var(--spectrum-alias-border-size-thin)
+    overflow: hidden;
+}
+.spectrum-Thumbnail-layer {
```

Comment:
Another new feature to make sure we're supporting?

Reply:
Yes this is part of a new variant, Layer which uses .spectrum-thumbnail-layer and `layer-inner` caught me mid way though getting this part implemented. It is in there now!